keyboards/doio/kb19/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

Comment:
This should be moved into the `default` keymap folder.

Reply:
Hi ,I've changed it . Thank you!